webpack/assets/javascripts/react_app/components/common/EmptyState/EmptyStatePattern.js

```diff
@@ -1,29 +1,50 @@
 import React from 'react';
-import { EmptyState as PfEmptyState } from 'patternfly-react';
+import { upperFirst } from 'lodash';
+import {
+  Title,
+  Button,
+  EmptyState,
+  EmptyStateVariant,
+  EmptyStateIcon,
+  EmptyStateBody,
+  EmptyStateSecondaryActions,
+} from '@patternfly/react-core';
+import * as pficons from '@patternfly/react-icons';
```

Comment:
This turns the bundle to be more than 4 mb!
we do not need to import all the pf icons here only for one usage.
IMO we should limit it to a  few opinionated icons or  we can use dynamic import.

Reply:
I will make make the required changed and update the pr.

Reply:
I think that for this PR it would be better to use the PF3 icons, since this case need to be generic..
until we will solve the dynamic loading issue or another way to load those icons efficiently,
I remember there were issues with webpack 3 and dynamic loading - maybe we should re-try when moving to webpack 4
cc @sharvit @tbrisker

Reply:
any more thoughts about this?

Reply:
@amirfefer @laviro I have made changes to use pf3 icons, let me know what you think!